src/openrct2/audio/AudioMixer.h

```diff
@@ -26,7 +26,7 @@ extern "C" {
 
 #define MIXER_VOLUME_MAX        128
 #define MIXER_LOOP_NONE         0
-#define MIXER_LOOP_INFINITE     -1
+#define MIXER_LOOP_INFINITE     (-1)
```

Comment:
What's the point of this?

Reply:
It prevents the `-` from being interpreted as the subtraction operator if the macro is used incorrectly, I guess.